internal/action/setup.go

```diff
@@ -137,6 +138,10 @@ func (s *Action) initGenerateIdentity(ctx context.Context, crypto backend.Crypto
 		}
 
 		email, err = termio.AskForString(ctx, "📧 What is your email?", email)
+
+		if strings.TrimSpace(email) == "" {
+			return fmt.Errorf("failed to create a usable key pair")
```

Comment:
Disclaimer: I'm not a maintainer. But I think this message could be more user-friendly. Something like: "You did not provide an email. An email is required to generate a gpg key pair, so please provide one."

Reply:
Done

Reply:
@jcrben  Can you please resolve this conversation?